frontend/src/app/messages/messages.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component } from '@angular/core';
+import { Http } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+
+import { environment } from '../../environments/environment';
+
+interface Message {
+  message: string;
+  showMessage: boolean;
+  type: string;
+}
+
+@Component({
+  selector: 'volontulo-messages',
+  templateUrl: './messages.component.html',
+})
+export class MessagesComponent {
+  messages$: Observable<Message[]>;
+  showMessages = true;
+
+  constructor(private http: Http) {
+    setTimeout(() => this.showMessages = false, 25000);
```

Comment:
I guess setTimeout could be initiated when the observable is resolved, not in component constructor directly.